app/liquid/liquid_renderer.rb

```diff
@@ -11,7 +11,9 @@ def initialize(page, layout: nil, request_country: nil, member: nil, url_params:
   end
 
   def render
-    template.render( data ).html_safe
+    Rails.cache.fetch(cache_key) do
+      template.render( data ).html_safe
+    end
```

Comment:
@osahyoun what behavior should we expect out of this in development? how are we setting which environments to cache in?